site/docs/.vitepress/config.ts

```diff
@@ -22,24 +22,17 @@ export default defineConfig({
     siteTitle: "grammY",
     externalLinkIcon: true,
     search: {
-      provider: "local",
+      provider: "algolia",
       options: {
+        apiKey: "33782ffb584887e3b8cdf9e760ea8e60",
+        indexName: "grammy",
+        appId: "RBF5Q0D7QV",
```

Comment:
Wouldn't it be better to hide these two tokens? It could also educate others to not write credentials in the code directly.

Reply:
No. Those values are delivered to the client. They are publicly available. It is not possible to hide them.